src/instructlab/lab.py

```diff
@@ -526,7 +547,12 @@ def generate(
 @click.option(
     "-m",
     "--model",
-    help="Model to use",
+    help="Model name to print in chat process",
+)
+@click.option(
+    "--model-path",
+    default=None,
+    help="Model path to use when starting server",
```

Comment:
same as https://github.com/instructlab/instructlab/pull/1002/files#r1583411423

Reply:
ack, sure. This can be separate PR.